src/components/NcEmojiPicker/NcEmojiPicker.vue

```diff
@@ -393,6 +391,7 @@ export default {
 	background-color: var(--color-main-background) !important;
 	border: 0;
 	color: var(--color-main-text) !important;
+	display: flex !important;
```

Comment:
The component keeps overwriting `display: flex;` with `display: inline-block`, which breaks the scrolling. So I force it here for now.